esphome/components/lvgl/lvgl_esphome.cpp

```diff
@@ -0,0 +1,118 @@
+#include "esphome/core/defines.h"
+#ifdef USE_LVGL
+#include "esphome/core/log.h"
+#include "esphome/core/helpers.h"
+#include "esphome/core/hal.h"
+#include "lvgl_hal.h"
+#include "lvgl_esphome.h"
+
+static const char *const TAG = "lvgl";
+
+namespace esphome {
+namespace lvgl {
```

Comment:
This should be inside the namespace really.
The functions outside can access it using `esphome::lvgl::TAG` correct?
```suggestion
namespace esphome {
namespace lvgl {

static const char *const TAG = "lvgl";
```